### ISSUE TITLE
broke it; can't seem to get dashboards provioned?

### ISSUE THREAD

**Issue by miekg** [spartanaorg/grafana#7]
I'm trying to see if the thing still works, but I can't seems to provision a dashboard. I'm also having trouble finding how to increase the log level. This (if I'm right) points to 2 things
1. there is little indication from the logs, which should be fixed, to add logging in this area
2. the fancy logging doesn't add that much can could be killed as well.

**Comment by miekg** [comment]
OK, that dashboard is picked up, but because user don't actually exist, or something it's not shown in the UI.

**Comment by miekg** [comment]
ok, this has def. to do with the user management. I tracked down the SQL query that gets executed for searching the dashboards:
~~~
 SELECT
			dashboard.id,
			dashboard.uid,
			dashboard.title,
			dashboard.slug,
			dashboard_tag.term,
			dashboard.is_folder,
			dashboard.folder_id,
			folder.uid as folder_uid,
			folder.slug as folder_slug,
			folder.title as folder_title
		FROM ( SELECT dashboard.id FROM dashboard  WHERE  dashboard.org_id=? AND
	(
		dashboard.id IN (
			SELECT distinct DashboardId from (
				SELECT d.id AS DashboardId
					FROM dashboard AS d
					LEFT JOIN dashboard AS folder on folder.id = d.folder_id
					LEFT JOIN dashboard_acl AS da ON
						da.dashboard_id = d.id OR
						da.dashboard_id = d.folder_id
					LEFT JOIN team_member as ugm on ugm.team_id = da.team_id
					WHERE
						d.org_id = ? AND
						da.permission >= ? AND
						(
							da.user_id = ? OR
							ugm.user_id = ? OR
							da.role IN (?)
						)
				UNION
				SELECT d.id AS DashboardId
					FROM dashboard AS d
					LEFT JOIN dashboard AS folder on folder.id = d.folder_id
					LEFT JOIN dashboard_acl AS da ON
						(
							-- include default permissions -->
							da.org_id = -1 AND (
							  (folder.id IS NOT NULL AND folder.has_acl = 0) OR
							  (folder.id IS NULL AND d.has_acl = 0)
							)
						)
					WHERE
						d.org_id = ? AND
						da.permission >= ? AND
						(
							da.user_id = ? OR
							da.role IN (?)
						)
			) AS a
		)
	) AND dashboard.folder_id IN (?) 
		ORDER BY dashboard.id  LIMIT 1000 OFFSET 0) as ids
		INNER JOIN dashboard on ids.id = dashboard.id
	
		LEFT OUTER JOIN dashboard folder on folder.id = dashboard.folder_id
		LEFT OUTER JOIN dashboard_tag on dashboard.id = dashboard_tag.dashboard_id ORDER BY dashboard.title ASC
~~~

**Comment by miekg** [comment]
ok, rolling back to an older commit does help. Inclined to revert a bunch. 

Much learned about the code, structure 'n stuff, so good progress: the user management runs deep

**Comment by miekg** [comment]
Fixed by #13

**Comment by miekg** [comment]
....and always test with a private browser session because otherwise your cookies will be taken as-is and you don't do the login flow.
